docs/errors.rst

```diff
@@ -125,21 +125,21 @@ Group 0 - Connection
 
 Related to the connection with the database.
 
-+---------------+-------------------------------------------------------------+
-| Code          | Description                                                 |
-+===============+=============================================================+
-| .. _pgrst000: | Could not connect with the database due to an incorrect     |
-|               | :ref:`db-uri` or due to the PostgreSQL service not running. |
-| PGRST000      |                                                             |
-+---------------+-------------------------------------------------------------+
-| .. _pgrst001: | Could not connect with the database due to an internal      |
-|               | error.                                                      |
-| PGRST001      |                                                             |
-+---------------+-------------------------------------------------------------+
-| .. _pgrst002: | Could not connect with the database when building the       |
-|               | :ref:`schema_cache` due to the PostgreSQL service not       |
-| PGRST002      | running.                                                    |
-+---------------+-------------------------------------------------------------+
++---------------+-------------+-------------------------------------------------------------+
+| Code          | HTTP status | Description                                                 |
++===============+=============+=============================================================+
+| .. _pgrst000: | 503, 504    | Could not connect with the database due to an incorrect     |
```

Comment:
Hm, IIRC `504` was added specially to handle the `db-pool-acquisition-timeout` error. Could we have a different PGRST error code for this case?